lib/utils/fetch.js

```diff
@@ -0,0 +1,26 @@
+const nodeFetch = require('node-fetch');
+const {version} = require('../../package.json');
+
+async function fetch(resource, options = {}) {
+	if (!options.headers) {
+		options.headers = {};
+	}
+	setUserAgent(options.headers);
+
+	return await nodeFetch(resource, options);
+}
+
+/**
+ * Always overwrite user agent to ours
+ * @param {Object} headers
+ */
+function setUserAgent(headers) {
+	const headerName = Object.keys(headers).find(headerName => headerName.toLowerCase() === 'user-agent');
+
+	headers[headerName || 'User-Agent'] = 'Suitest JS API v' + version;
```

Comment:
I would use common syntax for 'User-Agent': `<product>/<product-version>` where product is the same as name from package.json
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Also I would prefer concatanate user-agent current value, instead of overriding and rename function to `addUserAgent`

Reply:
changed header generation to `suitest-js-api/{version}`